app/kudos/migrations/0002_auto_20181016_1557.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1.1 on 2018-10-16 15:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kudos', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
We should delete migrations and recreate under one mig before pushing to staging @jasonrhaas